WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -391,6 +400,29 @@ private extension QuickStartTourGuide {
         NotificationCenter.default.post(name: .QuickStartTourElementChangedNotification, object: self, userInfo: [QuickStartTourGuide.notificationElementKey: QuickStartTourElement.noSuchElement])
     }
 
+    func setLocalNotification(for blog: Blog) {
+        guard let nextTour = tourToSuggest(for: blog) else {
+            return
+        }
+
+        QuickStartTourGuide.deletePendingLocalNotifications()
+
+        let content = UNMutableNotificationContent()
+        content.title = nextTour.title
+        content.body = nextTour.description
+        content.sound = UNNotificationSound.default
+        content.userInfo = [Notification.typeKey: Notification.local]
+
+        let interval = BuildConfiguration.current == .localDeveloper ? 30 : Constants.localNotificationInterval
+        let triggerDate = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second],
+                                                          from: Date(timeIntervalSinceNow: interval))
+        let trigger = UNCalendarNotificationTrigger(dateMatching: triggerDate, repeats: false)
```

Comment:
Logic looks good here, but is there a reason we're not using `UNTimeIntervalNotificationTrigger`?